openshift/templates/images/wordpress/README.md

```diff
@@ -0,0 +1,35 @@
+# WordPress Image
+
+[Back to images](../README.md)
+
+## Updating WordPress Core
+* Get [WordPress official image](https://hub.docker.com/_/wordpress) and verify.
+* Update [Image stream reference](https://github.com/bcgov/wordpress/blob/main/openshift/templates/dockerhub-imagestreams/wordpress.yaml#L36) to new version of the WordPress official image.  -- **Is this step needed**
+* Add new WordPress official image to OpenShift `oc process -f openshift/templates/dockerhub-imagestreams/wordpress.yaml | oc apply -f -` -- **Is this step needed**
```

Comment:
I assume these are to be updated once we have more information?

Reply:
I'm open to suggestions, but i think so, as there is additional information that will need to be changed as well